src/layout/App.tsx

```diff
@@ -16,8 +17,10 @@ import RouteUpdater from '../routes/RouteUpdater';
 
 import { getSessionExpiredAcknowledged, hideSessionExpiredDialog } from './appSlice';
 import AppHeader from '../features/header/AppHeader';
-import Intro from '../pages/Intro';
-import More from '../pages/More';
+import { ContentLoader } from '@rio-cloud/rio-uikit';
```

Comment:
To import a UIKIT component, please use the esm import via:
`import ContentLoader from '@rio-cloud/rio-uikit/lib/es/ContentLoader';`
in the future to tree-shake the UIKIT. Otherwise, the whole UIKIT is added to the production build :-) 

The route splitting is cool and is already used in other services. Thanks for adding that here too. 
I'd like to replace the ContentLoader with the Spinner to follow the UIKIT and other services.